src/UserService.Business/Commands/Education/CreateEducationCommand.cs

```diff
@@ -21,56 +32,112 @@ public class CreateEducationCommand : ICreateEducationCommand
     private readonly IAccessValidator _accessValidator;
     private readonly IHttpContextAccessor _httpContextAccessor;
     private readonly IDbUserEducationMapper _mapper;
+    private readonly IUserRepository _userRepository;
     private readonly IEducationRepository _educationRepository;
     private readonly ICreateEducationRequestValidator _validator;
+    private readonly IRequestClient<ICreateImagesRequest> _createImagesRequest;
+    private readonly ILogger<CreateEducationCommand> _logger;
 
     public CreateEducationCommand(
       IAccessValidator accessValidator,
       IHttpContextAccessor httpContextAccessor,
       IDbUserEducationMapper mapper,
+      IUserRepository userRepository,
       IEducationRepository educationRepository,
-      ICreateEducationRequestValidator validator)
+      ICreateEducationRequestValidator validator,
+      IRequestClient<ICreateImagesRequest> createImagesRequest,
+      ILogger<CreateEducationCommand> logger)
     {
       _accessValidator = accessValidator;
       _httpContextAccessor = httpContextAccessor;
       _mapper = mapper;
+      _userRepository = userRepository;
       _educationRepository = educationRepository;
       _validator = validator;
+      _createImagesRequest = createImagesRequest;
+      _logger = logger;
     }
 
-    public OperationResultResponse<Guid?> Execute(CreateEducationRequest request)
+    private async Task<List<Guid>> CreateImages(CreateEducationRequest request, List<string> errors)
     {
-      if (!_accessValidator.HasRights(Rights.AddEditRemoveUsers))
+      List<AddImageRequest> imagesToCreate = request.Images;
+      Guid userId = request.UserId;
+
+      if (imagesToCreate.Contains(null))
```

Comment:
why not just use request.Images and request.UserId? or you can add them both to method's parameters and don't create variables for them

Reply:
++